### PR TITLE
Add docs about integration status

### PR DESCRIPTION
**Pull request by wmora**
Feel free to suggest changes!

### CODE REVIEW

File: slate/source/includes/rest/core/_integrations.md [hunk 1]
```diff
@@ -645,6 +649,24 @@ Lists all integrations for a given app.
...
+| _id       | The integration ID.                                                                                             |
+| type      | The integration type. See [Integrations](/#integrations) for a list of supported types.                         |
+| status    | The integration status. Possible values are `active`, `inactive` or `error`. [More info](/#integration-status). |
+
+### Integration status
+
+| **Status** | **Description**                                                              |
+|------------|------------------------------------------------------------------------------|
+| active     | The integration is working correctly.                                        |
+| inactive   | There was a missing step while adding the integration from the Smooch dashboard. See [the docs](https://docs.smooch.io/guide/intro-to-channels/) for detailed steps on how to integrate a channel from the dashboard. |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_integrations.md]: Inactive integrations are not exclusive to the dashboard. A wechat integration created via the API is considered as inactive until we receive a webhook ping from wechat.

Comment (mspensieri) [reply slate/source/includes/rest/core/_integrations.md]: Maybe something like...

> The integration is incomplete. Additional action is required before this integration can be considered active

But I don't really like that

Comment (jpjoyal) [reply slate/source/includes/rest/core/_integrations.md]: Can we link to the rest doc instead ? https://docs.smooch.io/rest/#integrations

Can we give a couple example of what makes an integration inactive ? Could be a specific example for a channel in particular

Comment (wmora) [reply slate/source/includes/rest/core/_integrations.md]: > But I don't really like that

Too bad, I'll use it anyway 😆

**Pull request merged**